.gitignore

```diff
@@ -212,3 +212,5 @@ cython_debug/
 
 # End of https://www.toptal.com/developers/gitignore/api/flask,python
 
+# Personal ignores
+csn_project_db
```

Comment:
Hab die Datenbank von den Dateien in Git entfernt und bei .gitignore hinzugefügt. Diese Daten sind User-Spezifisch und werden in der Regel nicht in Git eingecheckt.